### PR TITLE
Add supported hardforks up to Merge.

### PR DESCRIPTION
**Pull request by jeffsmale90**
Adds support for the Berlin, London, Arrow Glacier, Gray Glacier and Merge hardforks.

Default is now "Merge" which is consistent with the `Ganache` CLI.

image

### CODE REVIEW

**Comment by jeffsmale90** [comment]
@davidmurdoch I assume that this sort order makes sense - newest to oldest?

File: src/integrations/ethereum/renderer/screens/config/ConfigScreens/ChainScreen.js [hunk 1]
```diff
@@ -116,15 +116,20 @@ class ChainScreen extends Component {
                 }
                 changeFunction={this.validateChange.bind(this)}
               >
+                <option value="merge">Merge</option>
```
Comment (MicaiahReid) [inline src/integrations/ethereum/renderer/screens/config/ConfigScreens/ChainScreen.js:119]: You're adding the Merge here, but that wasn't available until Ganache v7.7.0. UI is only upgraded to 7.5.0 :/

If it's easy to include an upgrade to latest that would be sweet! v7.7.3 has all of the latest hot fixes.

**Review by MicaiahReid — CHANGES_REQUESTED** [review]

File: src/integrations/ethereum/renderer/screens/config/ConfigScreens/ChainScreen.js (hunk 1 above)
Comment (jeffsmale90) [reply src/integrations/ethereum/renderer/screens/config/ConfigScreens/ChainScreen.js:119]: Oops - good catch!

I've upgraded to latest :)

**Comment by MicaiahReid** [comment]
:fire:

**Pull request merged**